contracts/libraries/IdToAddressBiMap.sol

```diff
@@ -26,6 +26,8 @@ library IdToAddressBiMap {
     }
 
     function insert(Data storage self, uint16 id, address addr) public returns (bool) {
+        require(addr != address(0), "Cannot insert zero address");
+        require(id != 2 ** 16 - 1, "Cannot insert max uint16");
```

Comment:
why not uint16(-1)

Reply:
Yes, if that is possible! For sure! Will do.